betelgeuse.py

```diff
@@ -299,7 +299,7 @@ def add_test_record(result):
 @click.option(
     '--jobs',
     '-j',
-    default=1,
+    default='1',
```

Comment:
Click will send the default value to the ParamType class assigned to the type. Instead of handling int->str->int conversion, is better to normalize the default to integer and just do one conversion str->int if needed.